Orange/widgets/utils/colorpalette.py

```diff
@@ -391,6 +392,12 @@ def colorSchemaChange(self):
 class ColorPalleteListing(QDialog):
     def __init__(self, parent=None, windowTitle="Color Palette List",
                  **kwargs):
+        # Import locally to prevent warning about using an obsolete module
+        # (This module itself is obsolete, but imported by its replacement,
+        # for backward compatibility, so importing colorbrewer globally would
+        # cause a warning.)
+        from Orange.widgets.utils import colorbrewer
```

Comment:
`colorbrewer` is still imported globally on line 23 and that seems fine as the replacement module (`colorpalettes`?) does not import this one.
I guess all of the above can be removed?